### PR TITLE
UI Profile Screen

### PR DESCRIPTION
**Pull request by Vanyaaafc**

### CODE REVIEW

File: lib/features/profile/presentation/profile_screen.dart [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            Text(
+              LocaleKeys.userName.tr(),
+              style: TextStyle(
+                fontSize: ScreenUtil().setHeight(16),
+                fontWeight: FontWeight.w500,
+              ),
+            ),
+            Text(
+              //will be dynamic
+              'username@gmail.com',
```
Comment (Andestrial123) [inline lib/features/profile/presentation/profile_screen.dart]: When?)

File: lib/features/profile/presentation/widgets/choose_language.dart [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+class ChooseLanguage extends StatefulWidget {
+  const ChooseLanguage({super.key});
+
+  @override
+  ChooseLanguageState createState() => ChooseLanguageState();
+}
+
+class ChooseLanguageState extends State<ChooseLanguage> {
+  String selectedLanguage = 'English';
```
Comment (Andestrial123) [inline lib/features/profile/presentation/widgets/choose_language.dart]: Move to conts

File: lib/features/profile/presentation/widgets/choose_language.dart [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    selectedLanguage = context.locale.languageCode == 'en' ? 'English' : 'Ukrainian';
```
Comment (Andestrial123) [inline lib/features/profile/presentation/widgets/choose_language.dart]: Same

File: lib/features/profile/presentation/widgets/choose_language.dart [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      },
+      child: Container(
+        padding: const EdgeInsets.symmetric(vertical: 12, horizontal: 16),
+        decoration: BoxDecoration(
+          color: selectedLanguage == language
+              ? CustomColors.whiteColor.withOpacity(0.2)
+              : Colors.transparent,
+          border: Border.all(
+            color: Colors.black,
+            width: selectedLanguage == language ? 2 : 1,
```
Comment (Andestrial123) [inline lib/features/profile/presentation/widgets/choose_language.dart]: What dose it meen 1 or 2 ?

File: lib/features/profile/presentation/profile_screen.dart (hunk 1 above)
Comment (Vanyaaafc) [reply lib/features/profile/presentation/profile_screen.dart]: done

File: lib/features/profile/presentation/widgets/choose_language.dart (hunk 4 above)
Comment (Vanyaaafc) [reply lib/features/profile/presentation/widgets/choose_language.dart]: delete this

File: lib/features/profile/presentation/widgets/choose_language.dart (hunk 2 above)
Comment (Vanyaaafc) [reply lib/features/profile/presentation/widgets/choose_language.dart]: impossible

File: lib/features/profile/presentation/widgets/choose_language.dart (hunk 3 above)
Comment (Vanyaaafc) [reply lib/features/profile/presentation/widgets/choose_language.dart]: impossible

**Review by Andestrial123 — APPROVED** [review]

**Pull request merged**